website/source/docs/provisioners/ansible.html.md

```diff
@@ -128,6 +128,11 @@ Optional Parameters:
 -   `user` (string) - The `ansible_user` to use. Defaults to the user running
     packer.
 
+-   `set_winrm_passwd` (boolean) - Set to `true` to access the randomly-generated 
+    EC2 Windows password through the environment variable `GENERATED_WINRM_PASSWORD`.
```

Comment:
I don't think this should mention EC2 or at least rephrase it a bit more generic. This could be any cloud provider that generate a random password.

Reply:
@rickard-von-essen I was only using the wording @swampdraggons used. https://github.com/hashicorp/packer/issues/5895

Reply:
Currently it only works for AWS, you can ignore my comment.